.travis.yml

```diff
@@ -3,9 +3,15 @@ osx_image: xcode10.1
 before_install:
   - cd Issue\ Manager
 install:
-  - echo 'placeholder'
+  - brew install git-lfs
+  - git lfs install
+  - git lfs version
+  - carthage version
 script:
-  - carthage bootstrap --platform iOS
-  - xcodebuild clean build -sdk iphonesimulator -project Issue\ Manager.xcodeproj/ -scheme Issue\ Manager CODE_SIGNING_REQUIRED=NO
+  - carthage bootstrap --platform iOS --verbose && xcodebuild clean build -sdk iphonesimulator -project Issue\ Manager.xcodeproj/ -scheme Issue\ Manager CODE_SIGNING_REQUIRED=NO
+  - sleep 10 # give logs time to catch up
```

Comment:
now everything is green; I'd consider removing all the `after_failure` and `sleep` commands, to keep the template clean.

Reply:
Good point. I still left the `sleep` in because output might still be truncated without it, but I removed the sanity checks.